docs/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## Next
+* Updated Select to take maxDropdownHeight prop to set maxHeight and overflow-y in cases where a scrolling list is desireable - implemented for `CompareLoadoutDrawer`
```

Comment:
Changelog is for user-visible changes, we don't document code changes here.

Reply:
Understood, will remove.